.github/workflows/tag-release.yaml

```diff
@@ -32,13 +32,22 @@ jobs:
           username: ${{ secrets.QUAY_USERNAME }}
           password: ${{ secrets.QUAY_PASSWORD }}
 
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Set up QEMU for multi-arch builds
+        uses: docker/setup-qemu-action@v2
+        with:
+          platforms: 'arm64'
```

Comment:
amd64 will be built by default??

Reply:
The host arch is amd64 so QEMU is not needed for that.